src/transactions/utils.ts

```diff
@@ -47,3 +54,42 @@ export function groupFeedItemsInSections(
       data: value.data,
     }))
 }
+
+export function* buildBaseTransactionReceipt(
+  receipt: TransactionReceipt | CeloTxReceipt,
+  networkId: NetworkId
+) {
+  const tokensById = yield* select((state) => tokensByIdSelector(state, [networkId]))
+
+  // Receipt gas information is based on the native token (there is no feeCurrency)
```

Comment:
I have tested the flow with contract kit and viem. And the receipt doesn't have any `feeCurrency` information, and checking the numbers with celo scan, the returned values represents the cost in Celo (I assume it's gonna be the native token for each chain). But we might want to revisit this logic in the future.

Reply:
Actually, I think this is a bug in celo scan. The `feeCurrency` is the one that we configure when we send the transaction. If you check how celo scan shows the transaction details:

<img width="730" alt="Screenshot 2023-11-24 at 16 36 14" src="https://github.com/valora-inc/wallet/assets/12504748/432614bb-4759-4bd5-a0fe-081497bedd72">
<img width="672" alt="Screenshot 2023-11-24 at 16 36 45" src="https://github.com/valora-inc/wallet/assets/12504748/b0caff67-4198-41a4-9510-fcd76ef8d8ae">

You can see that the cUSD transfers add up exactly to the transaction fee value.
The same with the gas price, it should be in cUSD.

So, I'm gonna re do this logic to make it right in a follow up PR.

Reply:
ooooh yes i remember this now, this is probably why we didn't implement the fees before.

is the proposal that we add the feeCurrency to the standby transaction, and then rely on the gas value in the receipt being for that feeCurrency? 

i think that we shouldn't merge any logic that we know is incorrect - it seems like assuming that the gas token is always the native token is only correct for Ethereum, so maybe we should only add fees for Ethereum right now and add a TODO for Celo (which i assume is the follow up PR that you mention). wdyt?

Reply:
Yes, I thought about this, I think the easiest solution would be to add the `feeCurrency` in the standby transaction. I don't like the fact that we will be storing a duplicated feeCurrency also in the `fees` to display it. But I think it's ok...

Yes, I'll merge this only for Eth, and I'll fix this flow for Celo in a next PR.